smartsim/_core/control/manifest.py

```diff
@@ -127,24 +112,16 @@ def _check_names(deployables: t.List[t.Any]) -> None:
     @staticmethod
     def _check_types(deployables: t.List[t.Any]) -> None:
         for deployable in deployables:
-            if not isinstance(deployable, (SmartSimEntity, EntitySequence)):
+            if not isinstance(deployable, (SmartSimEntity)):
```

Comment:
You do not need the extra () around SmartSimEntity

Reply:
changed